### DISCUSSION TITLE
Clarify idiomatic documentation in F# and Clojure

### DISCUSSION THREAD

**Discussion by ShalokShalom** [nxrighthere/UnrealCLR#64] [General]
Hi there :)

I am an F# developer and we usually use option types for error handling since this is one benefit of the functional way and fsharp in specific, you can read some of the benefits [here](https://stackoverflow.com/a/7960383/6712414) and also see the other arguments in that article, thanks a lot.

If you like to do it the old, traditional way, you can use [try/with](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/exception-handling/the-try-with-expression) instead, while the idiomatic way would be almost always be with option types. 

How can we address this, so this Readme reflects the idiomatic ways in both (all) languages?

I guess Clojure is supported too?

**Comment by nxrighthere** [comment]
Hey! Can you please provide a few examples of code from the plugin? I'm not familiar with F#, and since the entire framework written in C#, I don't have a clear conception about the subject.

As for Clojure, I never tried it personally, but it would be interesting to do some experiments with it.

**Reply by ShalokShalom** [reply]
I am actually not even using fsharp with Unreal right now, I just considered Unreal since it supports it obviously
